### PR TITLE
Add transaction history for lockbox partner

### PR DESCRIPTION
**Pull request by madisoncarr**
Hey Nicole, 
Here is the stuff that we worked on a couple weeks back.

### CODE REVIEW

File: app/models/lockbox_partner.rb [hunk 1]
```diff
@@ -20,4 +20,8 @@ def relevant_transactions_for_balance(exclude_pending: false)
     lockbox_action_ids = LockboxAction.excluding_statuses(excluded_statuses).pluck(:id)
     LockboxTransaction.where(lockbox_action_id: lockbox_action_ids)
   end
+
+  def historical_actions
+    @all_actions ||= lockbox_actions.order(:eff_date)
```
Comment (kyaroch) [inline app/models/lockbox_partner.rb]: It looks like what we want here is to order them from newest to oldest, which would be `.order(eff_date: :desc)`

**Comment by BintLopez** [comment]
Displaying the lockbox action history on the lockbox partner card and on its individual view...

image

image

**Comment by BintLopez** [comment]
Also, very apparent from this test data that the balance calculation is wrong 🤦‍♀ 💸

**Comment by BintLopez** [comment]
Follow up items
- [ ] work on the display for the transaction history (diff color for the support request line items, adding in the display text for the action types)
- [ ] fix the balance

**Pull request merged**